### PR TITLE
feat(kstepper): component reskin [KHCP-9001]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

[Figma](https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=2760%3A12829&mode=dev&t=m6UqtRoz81rfy7Vk-1)
[Jira](https://konghq.atlassian.net/browse/KHCP-9001)

`KStepper` component reskin

Links:

- [Kongponents sandbox](https://deploy-preview-2132--kongponents-sandbox.netlify.app/#/stepper)
- [Kongponents docs preview](https://deploy-preview-2132--kongponents.netlify.app/components/stepper.html)

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Tests coverage:** test coverage was added for new features and bug fixes
* [ ] **Docs:** includes a technically accurate README

### CODE REVIEW

File: docs/components/stepper.md [hunk 1]
```diff
@@ -14,145 +14,65 @@
...
+  { label: 'Pending step', state: 'pending' },
+  { label: 'Erroneous step', state: 'error' },
+  { label: 'Default step' }
+])
+
+const longSteps = ref<StepItem[]>([
+  { label: 'Step  a long long long long time ago', state: 'completed' },
+  { label: 'in a galaxy far far away', state: 'completed' },
+  { label: 'Kongponents were battling in space and', state: 'pending' },
+  { label: 'fighting robots and space monsters with lots of explosions' }
```
Comment (mptap) [inline docs/components/stepper.md:75]: 🤖 👾 💥 😄

**Review by mptap — DISMISSED** [review]
Functionally verified in sandbox, new Stepper looks 💅 ✨ 
Screenshot 2024-04-12 at 10 25 01 AM

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🚀 

I was really worried that my kongponents battle in space would be considered too unprofessional to get through the reskin. phew! 😌

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🚀

**Pull request merged**